_content/papers/devils-valley-to-omega-point_barrett.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Devil’s Valley to Omega Point: Reflections on the Emergence of a Theme from the Nō"
+authors: 
+  - "T.H. Barrett"
+editor: "Tadeusz Skorupski"
+external_url: "http://www.shin-ibs.edu/documents/bForum/v2/01Barrett.pdf"
+source: "https://www.shin-ibs.edu/research/archived-publications-and-research-projects/the-buddhist-forum/the-buddhist-forum-volume-ii/"
+drive_links:
+  - "https://drive.google.com/file/d/1wy-Q1ZEUY3h7_4FVxnETBfzlArM3P_Da/view?usp=share_link"
+formats: [pdf]
+course: nature
+tags:
```

Comment:
I think I'm using the comparative-studies tag only for the EBTs? And for how Chan Buddhism changed as it entered China, that's `chan-roots` 😁 So, let's go with:

```yaml
tags:
  - chan-roots
  - daoism
  - intercultural
```